app/policies/auth_app_login_option_policy.rb

```diff
@@ -0,0 +1,13 @@
+class AuthAppLoginOptionPolicy
```

Comment:
@monfresh These changes are all out of scope for this PR. I'm handling this refactoring in parallel work, as we already discussed.

Reply:
I disagree. See my comment in one of your refactoring PRs.